airflow/providers/databricks/provider.yaml

```diff
@@ -60,11 +60,13 @@ dependencies:
   # The 2.9.1 (to be released soon) already contains the fix
   - databricks-sql-connector>=2.0.0, <3.0.0, !=2.9.0
   - aiohttp>=3.6.3, <4
+  - databricks-sdk==0.10.0
```

Comment:
Why limit to specific version? we rarely do this

I also don't understand if this is dependency or extra dependency ?